client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -186,6 +186,10 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort }: useUsersManagem
         setFitlerRules(filterRulesToSet)
     }
 
+    useEffect(() => {
+        setPage(1);
```

Comment:
if the page is already 1, is the fetchUsers called?

Reply:
cause when I tried to do it it didnt work

Reply:
yes it is

Reply:
but isnt the fetch will be called twice? one time due to filter rules change and second time due to page change